src/engraving/layout/v0/tlayout.cpp

```diff
@@ -50,6 +50,7 @@
 #include "../libmscore/chord.h"
 #include "../libmscore/chordline.h"
 #include "../libmscore/clef.h"
+#include "../libmscore/capo.h"
```

Comment:
I think it would be preferable to do it in really correct alphabetical order; so `Capo` before `Chord` (in several other places too)

Reply:
I don't think it's a good idea. This approach adds a lot of complexity: every time you need to add a header here, you have to sort alphabetically by the following characters... We can split them into groups by the first character (as we do now), but add each new header at the bottom of each group